test/NuGet.Clients.Tests/NuGet.VisualStudio.Common.Test/OutputConsoleLoggerTests.Constructor.cs

```diff
@@ -12,6 +13,10 @@ public partial class OutputConsoleLoggerTests
     {
         public class Constructor : OutputConsoleLoggerTests
         {
+            public Constructor (GlobalServiceProvider sp)
+                : base(sp)
```

Comment:
What is use of passing `sp` here? Is it really used in base class other than resetting, not sure if it was necessary.